docs/tr/README-TR.md

```diff
@@ -0,0 +1,108 @@
+<!-- Deep Learning (with PyTorch)-->
+# Derin Öğrenme (PyTorch ile) [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/Atcold/pytorch-Deep-Learning/master)
+
+<!-- This notebook repository now has a [companion website](https://atcold.github.io/pytorch-Deep-Learning/), where all the course material can be found in video and textual format.
+-->
+
+Bu kurs deposunun materyallerini yazılı ve video halinde bulabileceğiniz [ilgili site](https://atcold.github.io/pytorch-Deep-Learning/).
```

Comment:
Here the link should point to the Turkish version, not the English one. Check the other translations. But yeah, at the moment there is no Turkish website yet. Still, let's fix this as a PR I will accept when there will be an `index-TR.md`, okay?